src/test/scala/org/broadinstitute/dsde/firecloud/integrationtest/IntegrationTestFixtures.scala

```diff
@@ -43,6 +43,7 @@ object IntegrationTestFixtures {
     Document(x._1, Map(
       AttributeName("library","datasetName") -> AttributeString(x._1),
       AttributeName("library","indication") -> AttributeString(x._2),
+      AttributeName("library","datasetOwner") -> AttributeString(x._2),
       AttributeName("library","numSubjects") -> AttributeNumber(x._3)
     ))
   }
```

Comment:
I'd suggest the following for readability:

```
val fixtureDocs:Seq[Document] = datasetTuples map { 
    case (access, phenotype, count) =>
      Document(access, Map(
        AttributeName("library","datasetName") -> AttributeString(access),
        AttributeName("library","indication") -> AttributeString(phenotype),
        AttributeName("library","datasetOwner") -> AttributeString(phenotype),
        AttributeName("library","numSubjects") -> AttributeNumber(count)
      ))
  }```

Reply:
good call, I've made this change - though we have the same pattern in other test fixtures, and I did NOT fix there.